CHANGELOG.md

```diff
@@ -1,3 +1,181 @@
+5.6.0 (16/01/2022)
+
+Supported API versions:
+* PA, SPAR, VAULT, PUB, AXP, BPM, FPO - V3
+* FIAB - V1
+
+Summmary:
+* Supporting new functionalities in pa,spar,vault,pub,quant and fi.
+
+Functionality Additions:
+* New Parameters are added in the FI Request.
+* 413 status code is added to Quant Reponse.
+* 'Warnings' are added as an enhancement to quant unit status response object.
+* In the Error response object new fields - code and title are added.
+* Added new end point for SPAR Product to get component details by Id.
+
+Breaking changes:
+* No changes.
+
+Bug Fixes:
+* Removed requiredfield attribute for pagenumber field parameter in GetAllCalculations End Point for pa,spar,vault,pub and quant.
```

Comment:
Removed default values when the parameter field is required for lookup end points for pa,spar,vault and strategy document end points for optimizers.